extensions/wikia/Forum/ForumController.class.php

```diff
@@ -87,6 +99,7 @@ protected function redirectToIndex() {
 	protected function getTopicTitle() {
 		$text = $this->wg->Title->getText();
 		$topicTitle = Title::newFromText( $text );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$topicTitle". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)